Cargo.toml

```diff
@@ -13,8 +13,10 @@ repository = "https://github.com/bytebeamio/rumqtt"
 license = "Apache-2.0"
 authors = ["tekjar <raviteja@bytebeam.io>"]
 
-[profile.dev.build-override]
-debug = true
+[profile.dev]
+# producing debug info in builds takes up a lot of space on runner, and we
+# are currently not using it.
```

Comment:
this doesn't only affect the ci / runners. it will affect the local builds as well.

ideally we should have separated this change from only CI specific change.